vignettes/articles/hub-validations-class.Rmd

```diff
@@ -40,22 +40,28 @@ The super class returned in each element depends on the status of the check:
 
 - If a check is skipped, a `<message/check_info>` condition class object is returned.
 
-- Checks vary with respect to whether they return an `<error/check_error>` or `<warning/check_failure>` condition class object if the check fails. 
-Ultimately, both will cause overall validation to fail and the two classes are used primarily to communicate the severity of a failing check.
+- Checks vary with respect to whether they return an `<error/check_error>` or `<error/check_failure>` condition class object if the check fails. 
+  - **`<error/check_error>` ** class objects indicate early termination of the validation process because of failure of a check downstream checks depend on.
+  - **`<error/check_failure>`** class objects indicate a check that failed but the failure does not affect downstream checks so validation was able to proceed.
+  
+Ultimately, both will cause overall validation to fail and the two classes are used primarily to communicate early termination.
 
 ## `hub_validations` print method
 
 `hub_validations` objects have their own print method which displays the result, the file name and message of each check:
 
 - `✔` indicates a check was successful (a `<message/check_success>` condition class object was returned)
-- `✖` indicates a high severity check failed (a `<error/check_error>` condition class object was returned)
-- `!` indicates a lower severity check failed (a `<warning/check_failure>` condition class object was returned)
+- `ⓧ` indicates a check that downstream checks depend on failed causing early return. (a `<error/check_error>` condition class object was returned)
+- `✖` indicates a check failed but does not affect downstream checks so validation was able to proceed (a `<error/check_failure>` condition class object was returned)
```

Comment:
```suggestion
- `✖` indicates a check with no downstream checks failed. (a `<error/check_failure>` condition class object was returned)
- `ⓧ` indicates a check failed and preventing downstream checks from being performed, causing early return. (a `<error/check_error>` condition class object was returned)
```

Reply:
Reordered and rephrased here: be60ccb4806082f3ebbdbed3eb875eae7ef16183